kivy/uix/behaviors/compoundselection.py

```diff
@@ -662,6 +667,7 @@ def select_node(self, node):
             nodes.append(node)
         self._anchor = node
         self._last_selected_node = node
+        self._reslove_last_node()
```

Comment:
Errors. Should be "self.resolve_last_node()"

Reply:
done